docs/_static/theme_overrides.css

```
@@ -28,3 +28,8 @@
  .rst-content #contents.hidden-first-item ul li li li {
    list-style: circle;
  }
+
+/* push down the RTD banner so it's not confused with menu */
+.keep-us-sustainable {
+  margin-top: 20em !important;
+} 
```

Let's add a newline at the end of the file
```suggestion
} 

```